packages/cypher-builder/src/expressions/procedures/db.ts

```diff
@@ -35,14 +35,15 @@ export class FullTextQueryNodes extends Clause {
     private targetNode: NodeRef;
     private indexName: string;
     private phrase: Variable;
-
+    private scoreVar: Variable | undefined;
     private whereClause: Where | undefined;
 
-    constructor(targetNode: NodeRef, indexName: string, phrase: Variable, parent?: Clause) {
+    constructor(targetNode: NodeRef, indexName: string, phrase: Variable, parent?: Clause, scoreVar?: Variable) {
```

Comment:
Usually, parent is better as the last parameter.

```suggestion
    constructor(targetNode: NodeRef, indexName: string, phrase: Variable, scoreVar?: Variable, parent?: Clause) {
```